packages/reports/tests/acceptance/navi-report-test.js

```diff
@@ -1752,4 +1752,39 @@ module('Acceptance | Navi Report', function(hooks) {
       'When not loading a report, the standard footer buttons are available'
     );
   });
+
+  test('Recreating same report after revert runs', async function(assert) {
+    await visit('/reports/2/view');
+
+    const columns = () => findAll('.table-widget__table-headers .table-header-cell').map(el => el.textContent.trim());
+    assert.deepEqual(columns(), ['Date', 'Property', 'Ad Clicks', 'Nav Clicks'], 'Report loads with expected columns');
+
+    await click(
+      findAll('.checkbox-selector--metric .grouped-list__item')
```

Comment:
We could just use contains to select on the right metric 
`await click($('.checkbox-selector--metric .grouped-list__item:contains(Page Views) .grouped-list__item-label')[0]);`

Reply:
I'm totally over `contains`